src/modules/customers/dto/create-customer.dto.ts

```diff
@@ -25,4 +25,28 @@ export class CreateCustomerDto {
   @MinLength(8)
   @MaxLength(10)
   Company_Registration_Number__c: string;
+
+  @ApiProperty({ description: 'Credit Risk Rating' })
+  @IsString()
+  @IsNotEmpty()
+  CCM_Credit_Risk_Rating__c: string;
+
+  @ApiProperty({ description: 'Credit Risk Rating Date' })
+  @IsString()
+  @IsNotEmpty()
+  CCM_Credit_Risk_Rating_Date__c: string;
+
+  @ApiProperty({ description: 'Loss Given Default' })
+  @IsNumber()
+  @IsNotEmpty()
+  @Min(0)
+  @Max(100)
```

Comment:
Is the maximum `100`, want to make sure this can also be a float number or a string with percentage sign?

Reply:
I'm enforcing this to be a float/number not a string which I think is reasonable. That's how it's stored in DTFS anyway so seems to be the most consistent choice. Floats pass these conditons as to plain integers.